nornir/core/inventory.py

```diff
@@ -70,11 +70,11 @@ class Host(object):
     def __init__(self, name, groups=None, nornir=None, defaults=None, **kwargs):
         self.nornir = nornir
         self.name = name
-        self.groups = groups or []
+        self.groups = groups if groups is not None else []
```

Comment:
What am I missing, why was this done? Do we want groups to be able to be `False`?

Reply:
If the inventory creates a host passing an empty list of groups it has to be accepted. Otherwise, when the inventory adds a group later on the Host won't have access to it.